cranelift/codegen/src/isa/aarch64/lower_inst.rs

```diff
@@ -1412,6 +1412,8 @@ pub(crate) fn lower_insn_to_regs<C: LowerCtx<I = Inst>>(
                 ctx.emit(Inst::FpuCSel32 { cond, rd, rn, rm });
             } else if is_float && bits == 64 {
                 ctx.emit(Inst::FpuCSel64 { cond, rd, rn, rm });
+            } else if is_float && bits == 128 {
+                ctx.emit(Inst::VecCSel { cond, rd, rn, rm });
```

Comment:
Just to note, it is weird to see that `ctx.emit(Inst::CSel ...` below did not have assert to prevent float/128 to become csel.